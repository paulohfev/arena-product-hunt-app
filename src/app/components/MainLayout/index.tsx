'use client'

import { Wrapper } from './MainLayout.styled'

export type MainLayoutProps = {
  children: React.ReactNode
}

const MainLayout: React.FC<MainLayoutProps> = ({ children }) => {
  return <Wrapper>{children}</Wrapper>
}

export default MainLayout
