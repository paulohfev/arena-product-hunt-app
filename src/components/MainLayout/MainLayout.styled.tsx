import styled from 'styled-components'

export const Wrapper = styled.main`
  background-color: #f0f0f0;
  padding: 16px;
`
