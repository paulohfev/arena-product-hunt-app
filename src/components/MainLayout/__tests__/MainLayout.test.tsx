import { render } from '@testing-library/react'

import MainLayout from '../index'

describe('MainLayout', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <MainLayout>
        <div>Test Content</div>
      </MainLayout>
    )
    expect(container).toMatchSnapshot()
  })
})
