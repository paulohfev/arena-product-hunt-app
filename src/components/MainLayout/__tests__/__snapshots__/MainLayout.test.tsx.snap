// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout matches snapshot 1`] = `
<div>
  <main
    class="sc-cab4b9bd-0 igXLKM"
  >
    <div>
      Test Content
    </div>
  </main>
</div>
`;
